bindings/rust/evmc-declare/src/lib.rs

```diff
@@ -0,0 +1,310 @@
+#![recursion_limit = "128"]
```

Comment:
Why is this needed?

Reply:
The default recursion limit is `64`, which at the moment has not proven viable for parsing some structures in this macro.

Reply:
Can you leave this comment above the setting?